lms/templates/courseware/info.html

```diff
@@ -38,12 +38,12 @@ <h2 class="title">${_("You are not enrolled yet")}</h2>
 
 <%include file="/courseware/course_navigation.html" args="active_page='info'" />
 
-<%static:require_module module_name="js/courseware/toggle_element_visibility" class_name="ToggleElementVisibility">
+<%static:require_module_async module_name="js/courseware/toggle_element_visibility" class_name="ToggleElementVisibility">
```

Comment:
If this used a factory then it should be included on the page inside a block, and then it wouldn't need to be async...